addons/ai/fnc_taskAttack.sqf

```diff
@@ -31,4 +31,6 @@ params ["_group","_position", ["_radius",0]];
 _group = _group call CBA_fnc_getGroup;
 if !(local _group) exitWith {}; // Don't create waypoints on each machine
 
+[_group] call CBA_fnc_clearWaypoints; // Clear existing waypoints

```

Comment:
`// Clear existing waypoints` tells the reader _what_ the line does, not _why_ it is being done. (and it feels a bit redundant, since the name of the function being called hints at that).